Introduce versioned addons and add-on removal, fixes #3555, fixes #3919 (#4891)
ddev/ddev: docs/content/users/usage/commands.md

# Copy an add-on from a tarball in another directory
ddev get /path/to/tarball.tar.gz

# View installed add-ons
ddev get --installed

# Remove an add-on
ddev get --remove redis
```

> In general, you can `ddev get` an add-on multiple times without doing any damage; Updating an add-on can be done with just `ddev get <add-on-name>`. If you have changed an add-on file and removed the `#ddev-generated` marker in the file, that file will not be touched and DDEV will let you know about it.

Reviewer: Run commands, don’t treat `ddev get` as a verb where we’re explaining what it does. Fix punctuation. Avoid “just” again.

```suggestion
In general, you can run `ddev get` multiple times without doing any damage. Updating an add-on can be done by running `ddev get <add-on-name>`. If you have changed an add-on file and removed the `#ddev-generated` marker in the file, that file will not be touched and DDEV will let you know about it.
```

Author: I just don't seem to be able to.

> In general, you can `ddev get` an add-on multiple times without doing any damage; Updating an add-on can be done with just `ddev get <add-on-name>`. If you have changed an add-on file and removed the `#ddev-generated` marker in the file, that file will not be touched and DDEV will let you know about it.

Reviewer: Run commands, don’t treat `ddev get` as a verb where we’re explaining what it does. Fix punctuation. Avoid “just” again.

```suggestion
In general, you can run `ddev get` multiple times without doing any damage. Updating an add-on can be done by running `ddev get <add-on-name>`. If you have changed an add-on file and removed the `#ddev-generated` marker in the file, that file will not be touched and DDEV will let you know about it.
```
